alluxio-ui/common/src/components/Header/Header.tsx

```diff
@@ -32,7 +32,7 @@ import {isExternalLink} from '../../utilities';
 
 import './Header.css';
 
-interface IHeaderProps {
+export interface IHeaderProps {
```

Comment:
Just curious, we need to export now because these are used in the tests?

Reply:
affirmative